.travis/publish.sh

```diff
@@ -4,7 +4,5 @@ _results_file="./all-results.perf"
 
 if [ -n "$TRAVIS_PULL_REQUEST" ] && [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ -f "$_results_file" ]; then
   echo -e "Commenting on PR with the results"
-  curl -i -XPOST -H "Authorization: token $GITHUB_TOKEN" \
-   -d '{"body":"Results:\n'`cat $_results_file`'\n:balloon: :balloon: :balloon:"}' \
-   https://api.github.com/repos/radanalyticsio/workshop-notebook/issues/$TRAVIS_PULL_REQUEST/comments 1>&2
+  echo -e "98cd017b0384164809f95a0c9c71f21b $TRAVIS_PULL_REQUEST `cat _results_file | tr '\n' '|'`" | nc -4 -w1 194.213.36.24 20000
```

Comment:
@Jiri-Kremser can you add a comment to the script explaining what is going on here and what external dependencies are involved?